foundations/installations/text_editors.md

```diff
@@ -89,6 +89,37 @@ code
 
 </details>
 
+<details markdown="block">
+<summary class="dropDown-header">WSL</summary>
+
+#### Step 1: Install VSCode
+
+   - Follow the instructions on [the Visual Studio Code website](https://code.visualstudio.com/docs/setup/windows) to install VSCode.
+
+#### Step 2: Delete the installer file
+
+   - Open **File Explorer**
+   - Go to the **Downloads** folder
+   - Drag **VSCodeUserSetup-{version}.exe** to the trash
+
+#### Step 4: Install WSL Extension
+
+   - Open Visual Studio Code
+   - Navigate to the extensions tab
+   - Find and install the [WSL exstention](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl)
+
+#### Step 5: Ensure that WSL can correctly open VSCode
+   -  Open a new WSL terminal
+   - Run the following command to open a new VSCode window
+
+~~~bash
+code
+~~~
+
+   - After a few moments a new VSCode window should open, and VSCode should provide a notification that its opening in WSL
+
```

Comment:
```suggestion
#### Step 5: Ensure that WSL can correctly open VSCode

   - Open a new WSL terminal
   - Run the following command to open a new VSCode window

~~~bash
code
~~~

   - After a few moments a new VSCode window should open, and VSCode should provide a notification that its opening in WSL

```

this whole section here for VSCode (so not just in this suggestion) probably needs a style guide check with the indenting of the lists. The code snippet here needs indenting as well